src/SdkCommon/ClientRuntime/ClientRuntime/IRestErrorModel.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
```

Comment:
missing xml comment avoids compiler warnings (we are already have more than 1000 warnings in our repo)
Please describe the interface and the purpose of the interface.